cypress/page/payment-step.page.ts

```diff
@@ -15,11 +15,8 @@ class PaymentStep{
         cy.get(this.proceedToCheckoutButton).click()
     }
 
-    public getConfirmationMessage(): void {
-        cy.get("#center_column > div > p > strong").should(
-            "have.text",
-            "Your order on My Store is complete.",
-        );
+    public getConfirmationMessage(): any {
+        return cy.get("#center_column > div > p > strong");
```

Comment:
you can create a class variable to store the selector used in this method, in a similar way that the other methods uses their selectors